src/components/Editor/DebugLine.js

```diff
@@ -70,7 +71,11 @@ export class DebugLine extends Component<Props, State> {
       return;
     }
 
-    const { location, location: { sourceId } } = selectedFrame;
+    const isGeneratedSource = !isOriginalId(selectedLocation.sourceId);
+    const location = isGeneratedSource
+      ? selectedFrame.generatedLocation || selectedFrame.location
+      : selectedFrame.location;
+    const sourceId = location.sourceId;
```

Comment:
this should work, but i think it would be better in the long run to keep our components dumb - and not let them know about this.

One good reason for this is because when you jump between original and generated contexts, it's good for the component to see a new location so it's updated